src/image_bios.cpp

```diff
@@ -5,10 +5,10 @@
 
 #include "config.h"
 
-#include "image_bios.hpp"
-
 #include "dbus.hpp"
 #include "fwupderr.hpp"
+#include "image_bios.hpp"
```

Comment:
`clang-format` on my machine moves this line to the previous position and I find it right.
Could you please revert this header position?

Reply:
Maybe it's time to update your local clang-format? =)
This is the second or third time I've had to manually fix the clang format because you have an old version.

Reply:
Yes, I remember it too, and now I use `clang-format version 12.0.1 (Fedora 12.0.1-1.fc34)`.
And one more thing, your new `src/nvm_x722.cpp` has commonly accepted order of the includes.
But in this case, it's broken. I don't know why, but guess it should be examined at your side. ;)

Reply:
In both cases I have the order specified by the project's [.clang-format](https://github.com/YADRO-KNS/obmc-yadro-fwupdate/blob/49017cae385ebfa0c2cac80b86e793eb14fdaf7d/.clang-format#L59):
```
 IncludeCategories:
  - Regex:           '^[<"](gtest|gmock)'
    Priority:        7
  - Regex:           '^"config.h"'
    Priority:        -1
  - Regex:           '^".*\.h"'
    Priority:        1
  - Regex:           '^".*\.hpp"'
    Priority:        2
  - Regex:           '^<.*\.h>'
    Priority:        3
  - Regex:           '^<.*\.hpp>'
    Priority:        4
  - Regex:           '^<.*'
    Priority:        5
  - Regex:           '.*'
    Priority:        6
```

nvm_x722.hpp:
```
#include "nvm_x722.hpp"

#include <fcntl.h>
#include <sys/mman.h>
#include <sys/stat.h>
#include <unistd.h>

#include <system_error>
```

image_bios.cpp:
```
#include "config.h"

#include "dbus.hpp"
#include "fwupderr.hpp"
#include "image_bios.hpp"
#include "nvm_x722.hpp"
#include "subprocess.hpp"
#include "tracer.hpp"

#include <fcntl.h>
#include <linux/i2c-dev.h>
#include <linux/i2c.h>
#include <sys/ioctl.h>
#include <unistd.h>

#include <chrono>
#include <fstream>
#include <thread>
```

It looks correct, isn't it?